code/stable/ssp/SRS/SSP_SRS.tex

```diff
@@ -389,8 +391,8 @@ \subsubsection{Theoretical Models}
                    \\ \midrule \\
                    Description & \begin{symbDescription}
                                  \item{${F_{S}}$ is the factor of safety (Unitless)}
-                                 \item{$P$ is the resistive shear force per meter in the z-direction ($\frac{\text{N}}{\text{m}}$)}
-                                 \item{$S$ is the mobilized shear force per meter in the z-direction ($\frac{\text{N}}{\text{m}}$)}
+                                 \item{$P$ is the resistive shear force ($\frac{\text{N}}{\text{m}}$)}
+                                 \item{$S$ is the mobilized shear force ($\frac{\text{N}}{\text{m}}$)}
```

Comment:
Why is per meter in the z-direction removed?

Reply:
This text comes from the short-name for the Unital. If I include the "per meter..." part in the short name, then it will show up everywhere (in these descriptions but also when the unital is referenced in assumptions, derivations, etc.). I originally included the "per meter..." part in the short name so that we would get that text in the descriptions, but when I noticed how it was changing everything else I changed it back. Having that text showing up everywhere made some passages unnecessarily bulky/awkward, in my opinion, though if we are okay with that I can change it back.

Reply:
Note also that the "per meter..." information isn't lost because it will still show up in the longer definition in the Table of Symbols, and also the units here will show N/m.

Reply:
@bmaclach, don't make any changes.  We'll review the whole thing again when it is complete and make sure everything "hangs" together.  The N/m part is something that I already think we might need to justify to a domain expert.  As I mentioned in the review, a domain expert will expect the force to be in N.